pkg/server/server.go

```diff
@@ -90,18 +90,21 @@ type Server struct {
 	PrometheusTenancyProxyConfig *proxy.Config
 	AlertManagerProxyConfig      *proxy.Config
 	MeteringProxyConfig          *proxy.Config
+	// Off cluster monitoring URLs.
+	OffClusterPrometheusURL   *url.URL
+	OffClusterAlertManagerURL *url.URL
 }
 
 func (s *Server) authDisabled() bool {
 	return s.Auther == nil
 }
 
 func (s *Server) prometheusProxyEnabled() bool {
-	return s.PrometheusProxyConfig != nil && s.PrometheusTenancyProxyConfig != nil
+	return (s.PrometheusProxyConfig != nil && s.PrometheusTenancyProxyConfig != nil) || s.OffClusterPrometheusURL.String() != ""
```

Comment:
Isn't the proxy config still set? Is this change needed?

Reply:
Yes, my bad ... fixing :+1: